test/fixtures.yml

```diff
@@ -16,6 +16,12 @@ adyen:
   merchant: merchant
   password: password
 
+agms:
+  login: 'osdgithub'
+  password: 'Ks1m32aF@'
+  api_key: 'accf69cefaeb1d19702e33b0a9bfc9f8f0ab3c065d937fc'
```

Comment:
Looks like `api_key` and `password` aren't needed, so please remove them from the fixture.